docs/modules/ROOT/pages/env_var_args.adoc

```diff
@@ -0,0 +1,56 @@
+
+=== PRODUCT_CONFIG
```

Comment:
Isn't this the same bunch of fields that we specify in `commandline_args.adoc`?

Reply:
yeah

Reply:
I'd merge them into one file then.

Reply:
I mean they come together via clap, the new opa side container e.g. does not use clap and would only require the env variable? I think in that case it makes sense to separate it?